ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/DefaultSynchronizer.java

```diff
@@ -132,7 +132,7 @@ private void startFullSync() {
 
   @Override
   public Optional<SyncStatus> getSyncStatus() {
-    if (!started.get()) {
+    if (!started.get() || syncState.isInSync()) {
```

Comment:
isInSync is true while we're within 5 blocks of head. Does that behaviour match this call or do we want to return something while we're closer to head?

Reply:
I believe what we need it to get the SyncResult and check if the distance to the head is equal to zero.

Reply:
modified to use if (syncState.syncStatus().getCurrentBlock() == syncState.syncStatus().getHighestBlock())